tensorflow_datasets/core/features/bounding_boxes.py

```diff
@@ -79,6 +82,38 @@ def encode_example(self, bbox):
         [bbox.ymin, bbox.xmin, bbox.ymax, bbox.xmax]
     )
 
+  def _build_thumbnail_with_bbox(self, ex: np.ndarray):
+    """Returns blank image with Bboxes drawn on it."""
+    PIL_Image = lazy_imports_lib.lazy_imports.PIL_Image
+    PIL_ImageDraw = lazy_imports_lib.lazy_imports.PIL_ImageDraw
+
+    SIZE = 150
+    blank_img = PIL_Image.new('RGB', (SIZE, SIZE), (255, 255, 255))
+    draw = PIL_ImageDraw.Draw(blank_img)
+
+    for i in range(ex.shape[0]):
+      # Rescale coordinates to match size of blank_image
+      ymin, xmin, ymax, xmax = ex[i, :]*SIZE
+      # Generate random rgb values for Bbox ouline
+      r, g, b = list(np.random.randint(0, 256, size=3))
```

Comment:
Color should be deterministic (generating the image twice should yield the same result). You should use https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.random.RandomState.html